### DISCUSSION TITLE
cannot delete a food that is no longer part of a recipe

### DISCUSSION THREAD

**Discussion by mlause** [TandoorRecipes/recipes#3144] [General]
Hello,

I deleted a recipe and want to delete some of the food items that were in that recipe. When I try to delete the food, it fails and a message says cannot delete an item that is part of a recipe. The recipe is gone. So I tried to do this with an existing recipe. I change an item out for the one I want to delete. It is not longer associated with that recipe, and I cannot delete it. In summary, the delete function doesn't work when a recipe is deleted or when an item is deleted from an existing recipe and not used anywhere else. 

This seems like a bug. 

M

**Comment by smilerz** [comment]
go to `/admin/cookbook/ingredient/` and `/admin/cookbook/step/` and run `delete xxx not part of a recipe`

As a quick & dirty method you can also merge the food with another food.

**Comment by mlause** [comment]
Thank you.
